### ISSUE TITLE
Not compatible with jython 2.7

### ISSUE THREAD

**Issue by Fyphon** [jsbruneau/jython-selfcontained-jar#1]
Not strictly a fault with this package but jython 2.7 has changed in a way that stops it working. 

BUILD SUCCESSFUL
Total time: 10 seconds
esmipau@elx56z7k32-yz:~/workspace/jython-selfcontained-jar$ java -jar dist/JythonExcelExample.jar 
Exception in thread "main" java.lang.VerifyError: Bad return type
Exception Details:
  Location:
    Main.newInterpreter(Z)Lorg/python/util/InteractiveConsole; @49: areturn
  Reason:
    Type 'org/python/util/JLineConsole' (current frame, stack[0]) is not assignable to 'org/python/util/InteractiveConsole' (from method signature)
  Current Frame:
    bci: @49
    flags: { }
    locals: { integer, 'java/lang/String' }
    stack: { 'org/python/util/JLineConsole' }
  Bytecode:
    0000000: 1a9a 000b bb00 0259 b700 03b0 b200 0412
    0000010: 0512 06b6 0007 4c2b b600 089e 000f 2bb8
    0000020: 0009 b600 0ac0 0002 b04d bb00 0c59 b700
    0000030: 0db0  
  Exception Handler Table:
    bci [30, 40] => handler: 41
  Stackmap Table:
    same_frame(@12)
    full_frame(@41,{Integer,Object[#41]},{Object[#42]})
    same_frame(@42)

```
at java.lang.Class.getDeclaredMethods0(Native Method)
at java.lang.Class.privateGetDeclaredMethods(Class.java:2615)
at java.lang.Class.getMethod0(Class.java:2856)
at java.lang.Class.getMethod(Class.java:1668)
at sun.launcher.LauncherHelper.getMainMethod(LauncherHelper.java:494)
at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:486)
```

**Comment by roelvanderplank** [comment]
Got a similar problem as well, with the (essentially identical) Main.java from https://wiki.python.org/jython/JythonFaq/DistributingJythonScripts.
Fixed it by simply changing the "return new JLineConsole();" with "return new InteractiveConsole();".

Yes, it really is that simple ;-0
Off course, as this is a point of the code that is only reached in some situations, it may be that 'my' code will fail on an unexpected point.
For me, however, this is now 'fixed'.

The only problem I myself have with this 'main' solution is that I need an extra argument.
Simply put, if my .py file needs arg1 and arg2, I need to call the 'output.jar' file with e.g. 'output.jar dummyarg arg1 arg2'.

No idea why, perhaps due to a simple 'start from 0/start from 1' mind between Java and Python.
Kind regards,
Roel
